### PR TITLE
getting rid of featured bar, it's a tag

### PR DESCRIPTION
**Pull request by ingawei**
Screen Shot 2022-10-31 at 1 02 19 PM
- also changing card style a little bit
- get rid of comment/tip icon for posts
- got rid of bounty flag, just don't think it's that important

### CODE REVIEW

**Comment by jahooma** [comment]
Thanks, it looks really nice!

What do you think about moving the "New" indicator into the same spot as Featured?

File: web/components/bet/quick-bet.tsx [hunk 1]
```diff
@@ -248,7 +248,7 @@ function BinaryQuickBetButton(props: {
       {hasInvestment && invested != null ? (
         <span
           className={clsx(
-            'text-sm font-light',
+            'my-auto text-sm font-light',
```
Comment (jahooma) [inline web/components/bet/quick-bet.tsx:251]: Could you instead use `items-center` on the parent Row?

File: web/components/contract/contract-card.tsx [hunk 2]
```diff
@@ -140,7 +141,7 @@ export function ContractCard(props: {
             <QuickOutcomeView contract={contract} />
           )}
         </Col>
-        <Row className={clsx('gap-1 px-4', children ? '' : 'mb-2')}>
+        <Row className={clsx('gap-1 px-4 ', children ? '' : 'mb-2')}>
```
Comment (jahooma) [inline web/components/contract/contract-card.tsx:144]: Extra space

File: web/components/contract/contract-details.tsx [hunk 3]
```diff
@@ -79,8 +76,6 @@ export function MiscDetails(props: {
           {'Resolved '}
           {fromNow(resolutionTime)}
         </Row>
-      ) : (contract.openCommentBounties ?? 0) > 0 ? (
```
Comment (jahooma) [inline web/components/contract/contract-details.tsx:82]: Thank you, I'm glad this is gone, haha!

**Review by jahooma — APPROVED** [review]
LGTM! Thanks for the improvements~

**Comment by ingawei** [comment]
> 
I could do that!

File: web/components/contract/contract-details.tsx (hunk 3 above)
Comment (ingawei) [reply web/components/contract/contract-details.tsx:82]: :)

**Pull request merged**